package.json

```diff
@@ -15,5 +16,9 @@
   "bugs": {
     "url": "https://github.com/dashevo/govobject-proposal/issues"
   },
-  "homepage": "https://github.com/dashevo/govobject-proposal#readme"
+  "homepage": "https://github.com/dashevo/govobject-proposal#readme",
+  "devDependencies": {
+    "bitcore-lib-dash": "github:dashevo/bitcore-lib-dash",
```

Comment:
Should we specify a tag/version here so that this version of govobject-proposal doesn't break if bitcore-lib-dash master is updated?